### ISSUE TITLE
[A11y][Contrast]Color contrast for the "Copy" button's background with page background ratio is 2.5:1 which is less than 3:1

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8195]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188278
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450) 

## Note:
Same issue is observed at Search button and for the stars in the table
## Repro Steps:

    Hit the URL and login with appropriate credentials
    Tab till "Profile" button in the menu bar and press enter
    Tab till any Manage packages"  and press enter.
    Tab till search bar and search for 'Json' and press enter 
    Tab till the first result "Newtonsoft.Json" and press enter 
    Tab till "Copy" button and  Check whether the color contrast for the background to icon is greater than equal to 3:1 or not

## Actual Results:

Color contrast for the background to focus on the icons Facebook and Twitter is which is less than 3:1.

   
## Expected Results:
Color contrast for the "Copy" button's background with page background ratio is 2.5:1 which is less than 3:1
## User Impact:
Users with low vision would face difficulty to identify the elements if the non-text contrast is less than 3:1

## Notes
This looks similar to https://github.com/NuGet/NuGetGallery/issues/8192
A global fix for this button type should be made.
